pkg/pillar/go.mod

```diff
@@ -206,3 +215,35 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace (
+	github.com/lf-edge/eve/pkg/kube/cnirpc => ../kube/cnirpc
```

Comment:
Do not bring back replace for cnirpc. Ever since it has been merged we do not need that anymore.

Reply:
Yeah this must be fixed.